### ISSUE TITLE
openjfx-web not supported?

### ISSUE THREAD

**Issue by FelipeAumannRS** [webfx-project/webfx#34]
Tried to load an url using WebEngine, although it does not work.

Interesting enough, after excluding webView from code and replacing it with any javafx ui component (let's say a label), and then running " mvn package", the index.html is automatically refreshed, showing my label in the browser.

It's said that WebFX has 80% compatibility with openjfx-web, although it's not possible to do anything with this dependency in a WebFX context.

**Comment by salmonb** [comment]
Yes, WebFX supports the JavaFX WebView and WebEngine, but the API coverage is not complete (estimated at 80%).

So there is probably something missing in your project.
Can you run `webfx update` at your project root level to see if that resolves the issue?

Don't hesitate to join the [WebFX Discord chat](https://discord.gg/xJcvmGvqh9) if you need quick support (GitHub didn't notify me about this issue for some reason)

**Comment by salmonb** [comment]
Closing due to inactivity. Please reopen if you can provide more details.
